tests/pl/climate_HassClimateSetTemperature.yaml

```diff
@@ -7,6 +7,7 @@ tests:
       name: "HassClimateSetTemperature"
       slots:
         temperature: 30
+    response: Ustawiono temperaturę na 30° stopni
```

Comment:
Czy tu ten symbol "°" powinien wciąż być?

Reply:
yes, note that the tests cover the temperature change with and without this symbol and this is intentional